### PR TITLE
Improved Speakers page

### PR DESCRIPTION
**Pull request by SparshBansal**
Improved Speaker Page . Resolved Issue #1078 . Please review and merge
@aayusharora Please check the links color and underlining and I have also fixed empty data issue
screenshot from 2017-02-26 07 59 42

### CODE REVIEW

**Comment by aayusharora** [comment]
@SparshBansal  please provide the live link. Thanks

**Comment by SparshBansal** [comment]
@aayusharora Here is the link : safe-sea-55983.herokuapp.com

**Comment by aayusharora** [comment]
@SparshBansal  It seem that you have understood the issue incorrectly, the underline and color should come on hovering the session.

**Comment by SparshBansal** [comment]
@aayusharora  Sorry i misunderstood , I have updated the PR and I request you to review and merge.
Now icon colors and underline change on hover.

Demo Link : https://safe-sea-55983.herokuapp.com/

File: src/backend/_scss/application.scss [hunk 1]
```diff
@@ -486,6 +486,11 @@ a {
   .pop>.pop-over.row {
 
     padding: 10px 20px 0 0;
+    &:hover{
+      .fa {
+        color: #000;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: @sparshBansal update it with color code from config.scss

File: src/backend/_scss/application.scss [hunk 2]
```diff
@@ -499,6 +504,13 @@ a {
 
     .pop-over>.sessions {
       padding: 0;
+      .sizeevent{
+        &:hover{
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: Nice .

**Pull request merged**